### PR TITLE
Added filtering to proto2gql

### PR DESCRIPTION
**Pull request by ziflex**
Added filtering of user's custom types.
As an addition, made some changes in proto2gql package layout (moved core logic into ```pkg``` folder).

### CODE REVIEW

File: cmd/proto2gql/main.go [hunk 1]
```diff
@@ -137,28 +147,78 @@ func main() {
...
+				// filter out matched types
+				return rPos.Match([]byte(typeName)) == false
+			})
+		}
+
+		if negative != "" {
+			rNeg, err := regexp.Compile(negative)
+
+			if err != nil {
+				panic("invalid regular expression: " + err.Error())
```
Comment (emicklei) [inline cmd/proto2gql/main.go:190]: please replace any panic with log.Fatalln

File: pkg/proto2gql/visitor.go [hunk 2]
```diff
@@ -63,10 +67,15 @@ func (v *Visitor) Flush(out io.Writer) {
 }
 
 func (v *Visitor) VisitMessage(m *proto.Message) {
-	v.buff.WriteString("\n")
-
+	// we add it any way just to bea ble to resolve it in fields
```
Comment (emicklei) [inline pkg/proto2gql/visitor.go]: be able

**Comment by ziflex** [comment]
Fixed

**Comment by emicklei** [comment]
thank you for contributing to this package.

**Pull request merged**